pkg/alertmanager/alerts/store.go

```diff
@@ -0,0 +1,64 @@
+package alerts
```

Comment:
Being the configdb store, may make sense to move it to `pkg/alertmanager/alerts/configdb/store.go` and just move the `AlertStore` interface back into `pkg/alertmanager/alerts`?

Reply:
Makes sense, I will also move the `pkg/alertmananger/local` package to be `pkg/alertmanager/alerts/local`